cmd/create_prs/create_prs_test.go

```diff
@@ -85,6 +85,27 @@ func TestItLogsCreatePrsSucceeds(t *testing.T) {
 	})
 }
 
+func TestItLogsCreateDraftPr(t *testing.T) {
+	fakeGitHub := github.NewAlwaysSucceedsFakeGitHub()
+	gh = fakeGitHub
```

Comment:
What is this variable needed for?  (and the one from line 92)

Reply:
both gh and g are global vars for the create_prs package.
They get invoked down from `runCommandDraft()` -> `cmd.Execute()` -> `run()`
https://github.com/Skyscanner/turbolift/blob/main/cmd/create_prs/create_prs.go#L77
https://github.com/Skyscanner/turbolift/blob/main/cmd/create_prs/create_prs.go#L92

Reply:
Not sure I'm a fan of injecting global variables from a test this way 😄 But it is the same as the other tests in this file anyway, so that's OK. I may go back and see if there's a different way to do this.